fury/tests/test_layout.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+import numpy.testing as npt
+from fury import actor
+from fury.layout import GridLayout
+
+
+def test_grid_layout_gets_cell_shape():
+    centers = np.asarray([[0, 0, 0], [0, 0, 0]])
```

Comment:
Strange, Could you explain why both center are at `(0,0,0)`

Reply:
There is no reason for selecting (0,0,0). I guess any center should do. Will change if needed.

Reply:
And, sorry for late response, had got a bit busy in coursework. :sweat_smile: